vars/main.yml

```diff
@@ -1,11 +0,0 @@
----
-
-# Default WATO host attributes. This variable, when being merged with
-# `checkmk_agent__host_attributes`, allows to successfully compare the WebAPI
-# get_host() response in case the `contactgroups` key is set.
```

Comment:
I see your point. But so far this is something that has many edge cases I still need to look into, so I'd prefer that you don't touch it.

The format of the server facts will change with debops-contrib/ansible-checkmk_server#53 anyway, so I'll have a look at the autojoin feature again in the foreseeable future. Then I'll make sure to clean it up and move it to `defaults`.